src/test/java/org/alliancegenome/curation_api/crud/controllers/DiseaseAnnotationITCase.java

```diff
@@ -55,7 +55,9 @@ public void createDiseaseAnnotation() {
         diseaseAnnotation.setCurie(DISEASE_ANNOTATION);
         diseaseAnnotation.setNegated(false);
         diseaseAnnotation.setObject(doTerm);
-        diseaseAnnotation.setSubject(biologicalEntity);
+        
+        // TODO fix due to SCRUM-510 refactor
+        //diseaseAnnotation.setSubject(biologicalEntity);
```

Comment:
@abecerra @markquintontulloch  do we need to check on these?

Reply:
That Integration test needs a change